src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -27,15 +28,22 @@ export default class AnnotationColumnFormatter
 
         if (copyNumberData)
         {
-            let oncoKbIndicator: IndicatorQueryResp|undefined;
-
-            if (oncoKbData && oncoKbData.result && oncoKbData.status === "complete") {
-                oncoKbIndicator = AnnotationColumnFormatter.getIndicatorData(copyNumberData, oncoKbData.result);
+            let oncoKbIndicator: IndicatorQueryResp|undefined = undefined;
+            let oncoKbStatus:"complete"|"pending"|"error" = "complete";
```

Comment:
we've got to have this in once place somewhere.  we use it so much.

Reply:
youre right